scripts/get_id_url.py

```diff
@@ -0,0 +1,27 @@
+"""Runs the webserver."""
+
+from absl import app
+from absl import flags
+from icubam import config
+from icubam.www import token
+from icubam.db import sqlite
+
+flags.DEFINE_string('config', 'resources/config.toml', 'Config file.')
+flags.DEFINE_string('dotenv_path', None, 'Optionally specifies the .env path.')
+flags.DEFINE_enum('mode', 'dev', ['prod', 'dev'], 'Run mode.')
+FLAGS = flags.FLAGS
+
+def main(argv):
+  cfg = config.Config(FLAGS.config, mode=FLAGS.mode, env_path=FLAGS.dotenv_path)
+  sqldb = sqlite.SQLiteDB(cfg.db.sqlite_path)
+  icus = sqldb.get_icus()
+  encoder = token.TokenEncoder(cfg)
+  for _, icu_row in icus.iterrows():
```

Comment:
check updater.py, that have function that generates the urls.
And we migrated to the new store:
```
for user in self.db.get_users():
  for icu in user.icu:
   ...
```

Reply:
Maybe I need to wait for the new db format ?
```
sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) no such column: users.icu_id
[SQL: SELECT users.icu_id, icu_name, name, users.telephone, description
           FROM users
           JOIN icus
           ON users.icu_id = icus.icu_id]
```

Reply:
It works, thanks to the new `icubam.db.store`